packages/expo/src/Expo.fx.tsx

```diff
@@ -0,0 +1,105 @@
+// load remote logging for compatibility with custom development clients
+import './environment/validate.fx';
+import './environment/logging.fx';
+import './environment/react-native-logs.fx';
+// load expo-asset immediately to set a custom `source` transformer in React Native
+import 'expo-asset';
+
+import { NativeModulesProxy, Platform } from '@unimodules/core';
+import Constants, { ExecutionEnvironment } from 'expo-constants';
+import * as Font from 'expo-font';
+import { installWebGeolocationPolyfill } from 'expo-location';
+import React from 'react';
+import { AppRegistry, StyleSheet } from 'react-native';
+
+import DevAppContainer from './environment/DevAppContainer';
+
+// install globals
+declare let module: any;
+// Represents an app running in the store client or an app built with the legacy `expo build` command.
+// `false` when running in bare workflow, custom dev clients, or `eas build`s (managed or bare).
+// This should be used to ensure code that _should_ exist is treated as such.
+const isManagedEnvironment =
+  Constants.executionEnvironment === ExecutionEnvironment.Standalone ||
+  Constants.executionEnvironment === ExecutionEnvironment.StoreClient;
+
+// Legacy convenience warning, this will be stripped in production.
+if (__DEV__) {
+  // @ts-ignore: TODO: not on the schema
+  if (Constants.manifest?.experiments?.redesignedLogBox) {
+    console.warn(
+      'LogBox is enabled by default on SDK 39 and higher. You can now remove the experiments.redesignedLogBox from your app configuration to get rid of this warning.'
+    );
+  }
+}
+
+// If expo-font is installed and the style preprocessor is available, use it to parse fonts.
```

Comment:
expo-font is always installed here

Reply:
I just added a comment so in the future if someone removes expo-font they'll wrap this in a try/catch (part of the motivation for this PR)